docs/api/index.rst

```diff
@@ -4,25 +4,25 @@
 API-specifications
 ==================
 
-The Objects API and Objecttypes API are scheduled to be a `recommended standard 
+The Objects API and Objecttypes API are scheduled to be a `recommended standard
 as of March 1, 2022`_. Their specifications can be found below.
 
 ======================  ==========================================
 API                     Specification version(s)
 ======================  ==========================================
-`Objecttypes API`_      2.0.0 (
-                        `Redoc <https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/maykinmedia/objecttypes-api/2.0.0/src/objecttypes/api/v2/openapi.yaml>`__,
-                        `Swagger <https://petstore.swagger.io/?url=https://raw.githubusercontent.com/maykinmedia/objecttypes-api/2.0.0/src/objecttypes/api/v2/openapi.yaml>`__
+`Objecttypes API`_      2.2.2 (
```

Comment:
[This comment](https://github.com/maykinmedia/objects-api/pull/514#discussion_r1906938031) remains unanswered here

Reply:
Fixed in https://github.com/maykinmedia/objects-api/pull/516/commits/bba5f3307140a5f2e2a2c6172c8725b23078b1e0